src/mgr/BaseMgrModule.cc

```diff
@@ -1540,6 +1546,9 @@ PyMethodDef BaseMgrModule_methods[] = {
   {"_ceph_unregister_client", (PyCFunction)ceph_unregister_client,
     METH_VARARGS, "Unregister RADOS instance for potential blocklisting"},
 
+  {"_ceph_get_daemon_health_metrics", (PyCFunction)ceph_get_daemon_health_metrics,
+    METH_VARARGS, "Get health metrics for all daemons"},
```

Comment:
If we have a new member of the Ceph-mgr Python API, let's give it the deserved Documentation welcome 😛